spring-cloud-kubernetes-client-discovery/src/main/java/org/springframework/cloud/kubernetes/client/discovery/KubernetesClientInformerSelectiveNamespacesAutoConfiguration.java

```diff
@@ -0,0 +1,166 @@
+/*
```

Comment:
this auto-configuration is only active when `ConditionalOnSelectiveNamespacesPresent` is satisfied, that is only when `spring.cloud.kubernetes.discovery.namespaces` has some value.